lib/remoteproc/rsc_table_parser.c

```diff
@@ -133,24 +133,22 @@ int handle_vdev_rsc(struct remoteproc *rproc, void *rsc)
 
 	/* only assign notification IDs but do not initialize vdev */
 	notifyid = vdev_rsc->notifyid;
-	if (notifyid == RSC_NOTIFY_ID_ANY) {
-		notifyid = remoteproc_allocate_id(rproc,
-						  notifyid, notifyid + 1);
+	notifyid = remoteproc_allocate_id(rproc,
+					  notifyid, notifyid + 1);
```

Comment:
looks like you have some white space changes, could you use tab for line alignment?